actions/actions.py

```diff
@@ -57,48 +149,45 @@ def backup() -> None:
     })
 
 
-def restore() -> None:
+def restore():
+    """
+    Implementation of easyrsa 'restore' action
+
+    Backup restoration process can be summarized as following:
+
+        * Selected backup is scanned and verified
+        * Contents of the backup are unpacked into <cahrm_dir>/EasyRSA/pki
+        * Data that are stored in the local database are updated
+        * All units that have relation with this easyrsa unit will be notified
+          about the certificate changes.
```

Comment:
Suggestion: document `:raises [ErrorType]: [ErrorDescription]` for the possible `RuntimeError` exception.